### ISSUE TITLE
Tooltip: XSS in target title

### ISSUE THREAD

**Issue by dgusoft** [primefaces/primefaces#5642]
## 1) Environment
- PrimeFaces version: 7.0.11
- Does it work on the newest released PrimeFaces version? No
- Does it work on the newest sources in GitHub? No
- Affected browsers: all

## 2) Expected behavior
Proper escaping of title referenced by p:tooltip for .

## 3) Actual behavior
Target title not escaped in tooltip.js if this.cfg.target is set in tooltip.js.

## 5) Sample XHTML
<p:inputText  id="focus" title="This tooltip is displayed when input gets the focus&lt;script&gt;alert('xss');&lt;/script&gt;" />
<p:tooltip id="toolTipFocus" for="focus" showEvent="focus" hideEvent="blur" />

## 6) Sample bean
-

## 7) Solution
This error was fixed for the bindGlobal function in tooltip.js with #4394. Same solution should be applied to Function bindTarget.
The line 
this.cfg.escape = (this.cfg.escape === undefined) ? true : this.cfg.escape;
must be added at top of bindTarget.

**Comment by melloware** [comment]
First trying your example I get this error:
```
javax.servlet.ServletException: javax.servlet.ServletException: Error Parsing /test.xhtml: Error Traced[line: 14] The value of attribute "title" associated with an element type "p:inputText" must not contain the '<' character.
```

But if I make it...

```xml
<p:inputText id="focus" title="This tooltip is displayed when input gets the focus&lt;script&gt;alert('xss');&lt;/script&gt;" />
<p:tooltip id="toolTipFocus" for="focus" showEvent="focus" hideEvent="blur" />
```

Then I can reprorduce it with this reproducer.
[pf-5642.zip](https://github.com/primefaces/primefaces/files/4287185/pf-5642.zip)

**Comment by dgusoft** [comment]
Sorry for the copy&paste error. Thanks for the quick error fix. Will the fix be ported to 7.0.x?

**Comment by melloware** [comment]
That is up to @mertsincan to port it to Elite.

**Comment by fbraun-etas** [comment]
Was the issue also fixed for the latest Elite version of 5.2.x?

**Comment by fbraun-etas** [comment]
Okay, so when will it be fixed for the latest Elite version of 5.2.x?

**Comment by melloware** [comment]
5.2 is 6 years old I doubt it will be patched.  Your better bet is to upgrade PrimeFaces.
